### ISSUE TITLE
Fix "Quickstart" example to use interactive HTML elements

### ISSUE THREAD

**Issue by jenweber** [ember-learn/guides-source#655]
In our [Quickstart example in the Guides](https://guides.emberjs.com/release/getting-started/quick-start/#toc_click-events), we are teaching people to add a click event to a list item, `<li>`, and it should be a `<button>` instead. However `<li>` are not an interactive element, which means that some browsers may not fire the event, it may not receive focus for keyboard nav, and it does not work well with assistive technology like screen readers.

Here's the section to fix. Refactor this to not use `<ul>` and `<li>`. Use an interactive element like `<button>` for the action.

```
<ul>
  {{#each this.people as |person|}}
    <li>{{person}}</li>
    <li {{action "showPerson" person}}>{{person}}</li>
  {{/each}}
</ul>
```

Please also read the writing that goes with it carefully, to see if it refers to the items as lists.

**Comment by mike-north** [comment]
Attempted to fix this with the smallest amount of change in #657.

```html
<ul>
	<li>
		<button>foo</button>
	</li>
</ul>
```
should be ok

**Comment by jenweber** [comment]
Closed by #657. Thank you, Mike!
